avm/utilities/pipelines/sharedScripts/Set-ModuleReadMe.ps1

```diff
@@ -254,6 +254,12 @@ function Set-DefinitionSection {
         $Properties.Keys | ForEach-Object { $Properties[$_]['name'] = $_ }
     }
 
+    # Error handling: Throw error if any parameter is missing a category
```

Comment:
Don't we have a Pester test for that already?

Reply:
Hey @eriqua, we have. I added it here explicitely because I've seen more than one contribution in the past where somebody created a contribution with a missing category which does NOT fail the readme script nor it's test and who didn't have workflows. I'd feel a lot safer if it would not be so damn easy to miss.